setup.py

```diff
@@ -24,5 +25,6 @@
           'testfixtures'
       ],
       zip_safe=False,
-      python_requires='>=3.6'
+      python_requires='>=3.6',
+      package_data={'u8timeseries': ['VERSION']}
```

Comment:
What is this line adding?

Reply:
setuptools doesn't include all files in the installed package by default (see https://packaging.python.org/guides/using-manifest-in/). Without this line, the VERSION file would not be available for access after installation. I chose this approach in favour of including a MANIFEST.in because it doesn't require an extra file.

Reply:
sounds good :)